docs/pod-identity-mode.md

```diff
@@ -43,7 +40,7 @@ Not all steps need to be followed on the instructions for the aad-pod-identity p
     az keyvault set-policy -n $KEYVAULT_NAME --certificate-permissions get --spn <YOUR AZURE USER IDENTITY CLIENT ID>
     ```
 
-1. Add a new `AzureIdentity` for the new identity to your cluster
+2. Add a new `AzureIdentity` for the new identity to your cluster
```

Comment:
we can keep `1.`, markdown should automatically assign incrementing numbers

Reply:
Updated! Interestingly this got auto-updated by the markdown plugin I've on vscode but changed it just use `1.` for all.